visualization/app/codeCharta/ui/legendPanel/legendPanel.component.html

```diff
@@ -25,19 +25,15 @@
 	<cc-map-color-picker map-color-for="outgoingEdge" label="Outgoing Edge"> </cc-map-color-picker>
 	<cc-map-color-picker map-color-for="incomingEdge" label="Incoming Edge"> </cc-map-color-picker>
 
-	<hr ng-show="$ctrl._viewModel.packageLists[0]" />
-	<div
-		class="legend-block"
-		ng-show="$ctrl._viewModel.packageLists"
-		ng-repeat="packageList in $ctrl._viewModel.packageLists track by $index"
-	>
+	<hr ng-if="$ctrl._viewModel.packageLists[0]" />
+	<div class="legend-block" ng-show="$ctrl._viewModel.packageLists" ng-repeat="packageList in $ctrl._viewModel.packageLists">
 		<img ng-src="{{::packageList.colorPixel}}" alt="" />
 		<span
 			class="marked-package {{ $first ? '' : 'set-left-margin' }}"
-			ng-repeat="item in packageList.markedPackages track by $index"
+			ng-repeat="item in packageList.markedPackages"
```

Comment:
```suggestion
			ng-repeat="item in packageList.markedPackages track by item.path"
```

Reply:
I see, thanks
I changed it to avoid full object stringifying